polymorphic/admin.py

```diff
@@ -465,7 +477,8 @@ def get_form(self, request, obj=None, **kwargs):
         kwargs.setdefault('form', self.base_form or self.form)
 
         # prevent infinite recursion in django 1.6+
-        if not self.declared_fieldsets:
+        if not hasattr(self, 'declared_fieldset') or \
```

Comment:
I guess this is a typoo, and should be `declared_fieldsets` with an s ?
It could be replaced with `if getattr(self, 'declared_fieldsets', None):` and catch the exists check and bool check in one go.